src/pages/resultsView/network/network.htm

```diff
@@ -89,13 +88,61 @@
 
 </script>
 
+<script>
+    function formSubmit(path, params, target, method) {
+        // method="smart" means submit with GET iff the URL wouldn't be too long
+
+        const form = document.createElement('form');
+        let computedMethod = method;
+        if (method === "smart") {
+            computedMethod = buildCBioPortalPageUrl(path, params).length > 1800 ? "post" : "get"; // use POST if URL will be too large for some browsers
+        }
+        form.setAttribute('method', computedMethod);
+        form.setAttribute('action', path);
+        if (target) {
+            form.setAttribute('target', target);
+        }
+
+        for (const key of Object.keys(params)) {
+            const hiddenField = document.createElement('input');
+            hiddenField.setAttribute('type', 'hidden');
+            hiddenField.setAttribute('name', key);
+            hiddenField.setAttribute('value', params[key]);
+            form.appendChild(hiddenField);
+        }
+
+        document.body.appendChild(form);
+        form.submit();
+    }
+</script>
 
 <script type="text/javascript">
     // This is for the moustache-like templates
     // prevents collisions with JSP tags
     _.templateSettings = {
         interpolate: /\{\{(.+?)\}\}/g
     };
+
+    // calls to network do must now include all case ids
+    // so we need to hijack click and form submit
+    $(document).ready(function(){
+        $("body").on("click","a",function(e){
+            e.preventDefault();
+
+            switch(this.innerText) {
+                case "GraphML":
```

Comment:
man this is really rough.... capturing *every* `<a>` click? Is it possible to make it more specific?

Reply:
it's only aesthetically unpleasing.   it's better to bind to all clicks than to have to worry about rendering timeline.  i think we should just leave it.  it's harmless.

Reply:
i can easily limit it to the netmsg div.  this is actually not different because jquery is doing the same operation on every click, but whatever

Reply:
the click delegation is fine, but how about using a class selector? "a" is so broad. How is it safe - wouldn't this apply always once this page is loaded?

Reply:
the issue is in case any future link shows up with innerText = "GraphML" its unlikely but it still makes me uneasy

Reply:
this is better.